tools/performance/engine-benchmarks/bench_download.py

```diff
@@ -621,8 +621,8 @@ def commit_to_str(commit: Commit) -> str:
 
 
 if __name__ == '__main__':
-    default_since = datetime.now() - timedelta(days=14)
-    default_until = datetime.now()
+    default_since: date = (datetime.now() - timedelta(days=14)).date()
```

Comment:
sneaky ;)

Reply:
That is a small fix to make this script usable in my systemd daily job.